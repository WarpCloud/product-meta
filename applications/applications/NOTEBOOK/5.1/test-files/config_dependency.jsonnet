{
  application_type: "NOTEBOOK",
  application_name: "notebook1",
  application_version: "5.1",
  user_config: {},
}
