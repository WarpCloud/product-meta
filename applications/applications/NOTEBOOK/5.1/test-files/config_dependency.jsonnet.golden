{
   "Customized_Namespace": "default",
   "Develop": false,
   "Transwarp_Install_ID": "",
   "instance_list": [
      {
         "annotations": {
            "application_name": "notebook1",
            "application_type": "NOTEBOOK"
         },
         "configs": {
            "notebook_cpu_limit": 2,
            "notebook_cpu_request": 0.10000000000000001,
            "notebook_memory_limit": 4,
            "notebook_memory_request": 1
         },
         "dependencies": [
            {
               "moduleName": "hdfs",
               "name": "hdfs"
            },
            {
               "moduleName": "yarn",
               "name": "yarn"
            },
            {
               "moduleName": "localcran",
               "name": "notebook1-localcran"
            }
         ],
         "instance_settings": { },
         "moduleName": "notebook",
         "name": "notebook1-notebook",
         "version": "5.1"
      },
      {
         "annotations": {
            "application_name": "notebook1",
            "application_type": "NOTEBOOK"
         },
         "configs": {
            "localcran_cpu_limit": 1,
            "localcran_cpu_request": 0.10000000000000001,
            "localcran_memory_limit": 2,
            "localcran_memory_request": 1
         },
         "instance_settings": { },
         "moduleName": "localcran",
         "name": "notebook1-localcran",
         "version": "5.1"
      }
   ],
   "instance_settings": { },
   "version": ""
}
